frontend/src/metabase/entities/databases/forms.js

```diff
@@ -284,9 +295,12 @@ function getEngineFormFields(engine, details, id) {
         initial: field.default,
         readOnly: field.readOnly || false,
         helperText: field["helper-text"],
+        visibleIf: field["visible-if"],
```

Comment:
just a matter of style. I see above is a diff of `field["visible-if"]` -> `field.visibleIf` and then an addition of the string accessor version. Is this a deliberate use of the two different styles?

Reply:
cc @ranquild

Reply:
`field` is raw data returned from the backend, that's why it's `visible-if`